### PR TITLE
Copy Codeblock Button

### PR DESCRIPTION
**Pull request by 8bitgentleman**
Adds a button to all codeblocks to let you copy their contents to the clipboard

### CODE REVIEW

**Comment by 8bitgentleman** [comment]
Anyone know why this build is failing? Says there's an illegal white space but doesn't say _where_ it is

File: extensions/8bitgentleman/copy-code-blocks.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Code Block Copy Button",
+    "short_description": "Adds a button in code blocks to copy their contents to the clipboard.",
+    "author": "Matt Vogel",
+    "tags": ["code block"],
+    "source_url": "https://github.com/8bitgentleman/roam-depot-copy-code-block",
+    "source_repo": "https://github.com/8bitgentleman/roam-depot-copy-code-block.git",
+    "source_commit": "8acd0b22e367dd36ab1909e1911ee37da7add906",
+    "stripe_account": "acct_1LJFEYQmxalymEZL"
+
```
Comment (panterarocks49) [inline extensions/8bitgentleman/copy-code-blocks.json:10]: Our json parser is not very forgiving, I think this line is the issue

Comment (8bitgentleman) [reply extensions/8bitgentleman/copy-code-blocks.json]: I deleted the empty line there but no change change, could it be something else?

Comment (panterarocks49) [reply extensions/8bitgentleman/copy-code-blocks.json]: Empty line is still there in files changed 
image

Comment (8bitgentleman) [reply extensions/8bitgentleman/copy-code-blocks.json]: ended up being a rogue `.DS_Store` file I think

**Review by panterarocks49 — APPROVED** [review]
Small bug, if you want to fix in another PR
If I edit the code block and then click copy, it doesn't copy the current text of the code block
This is because we only update our database on blur

Would love to have this for inline code as well!

**Pull request merged**